src/overlays/actors/ovl_En_Bubble/z_en_bubble.c

```diff
@@ -333,7 +333,7 @@ void EnBubble_Init(Actor* thisx, PlayState* play) {
     Collider_InitJntSph(play, &this->colliderSphere);
     Collider_SetJntSph(play, &this->colliderSphere, &this->actor, &sJntSphInit, this->colliderElements);
     CollisionCheck_SetInfo2(&this->actor.colChkInfo, DamageTable_Get(9), &sColChkInfoInit);
-    this->actor.hintId = 0x16;
+    this->actor.hintId = TATL_HINT_ID_IGOS_DU_IKANA;
```

Comment:
```suggestion
    //! @bug: hint Id not correctly migrated from OOT `NAVI_ENEMY_SHABOM`
    this->actor.hintId = TATL_HINT_ID_IGOS_DU_IKANA;
```
Can confirm, this is the wrong hint. Can we add a bug documentation for it?
![Screenshot_2023-11-14_18-06-52](https://github.com/zeldaret/mm/assets/42048411/ed121193-5079-4842-a03b-d19afbfb751e)

Reply:
It seems like this same bug is also present in `z_en_elfbub.c` (stray fairies in bubbles), I'll mark that bug as well